tests/acceptance/FormEditorCreateCustomFieldCest.php

```diff
@@ -30,11 +43,8 @@ public function createCustomSelect(\AcceptanceTester $i) {
     $i->fillField('[data-automation-id="custom_field_value_settings_value"]', 'First option'); // Configure first option
     $i->click('[data-automation-id="custom_field_values_add_item"]'); // Add second option
 
-    // Save custom field
-    $i->click('[data-automation-id="create_custom_field_submit"]');
-    $i->waitForText('Custom field saved', 10, '.automation-dismissible-notices');
-    $i->seeNoJSErrors();
-    $i->click('.automation-dismissible-notices .components-notice__dismiss'); // Hide notice
+    // Save the custom field
+    $this->saveCustomFieldBlock($i);
```

Comment:
I think this comment and maybe a couple of others are redundant since they actually just repeat what is already said in the name of the following method.
Also [recently we found out](https://mailpoet.slack.com/archives/CAUJT7D1Q/p1599647422007800) that there is a nicer way of adding comments into acceptance tests using `$i->wantTo('Do something');` so that the comment is also printed out on the output.
I think it would be nicer to remove redundant comments and replace others with `$i->wantTo('Do something');` type of comments. What do you think?

Reply:
Well, I don't think that's a good idea, because commenting is used for dividing the code between the notice.
With `$i->wantTo('Do something');`, the code will be all the same, hard to see commenting which is important when previewing the code, but on the other hand we would have the output. Not sure if we really need additional output, was able to see where failed and why.

Reply:
I agree with Rosta here. I think it is better to use the syntax `$i->wantTo('Do something');` you still see it in the code as comments and you can divide the code using empty lines if you want to.

Reply:
@veljkho Here is a sample of the change from the current PR:

Before:
```php
$i->wantTo('Create custom field: text input');
// Configure, check and save the custom field block
$i->waitForElement('[data-automation-id="create_custom_field_form"]');
$i->selectOption('[data-automation-id="create_custom_field_type_select"]', 'Text Input');
$i->fillField('[data-automation-id="create_custom_field_name_input"]', 'My custom text input');
$i->selectOption('[data-automation-id="settings_custom_text_input_validation_type"]', 'Numbers only');
$this->saveCustomFieldBlock($i);

// Save, reload and check data were saved
$i->saveFormInEditor();
$i->reloadPage();
$this->checkCustomTextInputInForm($i);


// Change text input validation
$i->selectOption('[data-automation-id="settings_custom_text_input_validation_type"]', 'Alphanumerical');
$i->click('Update custom field');
$i->waitForText('Custom field saved.');

// Update label and save the form
$i->fillField('[data-automation-id="settings_custom_text_label_input"]', 'My updated custom text input');
$i->saveFormInEditor();
```

After:
```php
$i->wantTo('Create custom field: text input');

$i->wantTo('Configure, check and save the custom field block');
$i->waitForElement('[data-automation-id="create_custom_field_form"]');
$i->selectOption('[data-automation-id="create_custom_field_type_select"]', 'Text Input');
$i->fillField('[data-automation-id="create_custom_field_name_input"]', 'My custom text input');
$i->selectOption('[data-automation-id="settings_custom_text_input_validation_type"]', 'Numbers only');
$this->saveCustomFieldBlock($i);

$i->wantTo('Configure, check and save the custom field block');
$i->saveFormInEditor();
$i->reloadPage();
$this->checkCustomTextInputInForm($i);

$i->wantTo('Change input validation');
$i->selectOption('[data-automation-id="settings_custom_text_input_validation_type"]', 'Alphanumerical');
$i->click('Update custom field');
$i->waitForText('Custom field saved.');

$i->wantTo('Update input label');
$i->fillField('[data-automation-id="settings_custom_text_label_input"]', 'My updated custom text input');

$i->saveFormInEditor();
```
I see your point that comments are better highlighted in a code editor. On the other hand, if you separate those blocks with empty lines I think it is not bad either plus you have a benefit of human-readable text on the output. I think seeing `'Change input validation'` in error output helps to find out what part of a test failed without having to see full source code. We lived with simple comments all the time so I guess it is not that important. I'll approve the PR and let you decide if you want to add the change.